src/pydantic_zarr/v3.py

```diff
@@ -948,6 +948,7 @@ def from_flat_group(
     # copy the input to ensure that mutations are contained inside this function
     data_copy = dict(data).copy()
     # Get the root node
+    print("hi")
```

Comment:
```suggestion
```

Reply:
🤦

Reply:
fwiw `print("hi")` is one of my favorite debug statements